core/core.go

```diff
@@ -70,15 +70,15 @@ func (h *Handler) info(ctx context.Context) (map[string]interface{}, error) {
 
 func (h *Handler) leaderInfo(ctx context.Context) (map[string]interface{}, error) {
 	var (
-		generatorHeight  *uint64
-		generatorFetched *time.Time
+		generatorHeight  uint64
```

Comment:
Are the SDK or dashboard currently looking for json `null` in this field? Do we need to update clients to look for `0` instead?